README.md

```diff
@@ -16,6 +16,9 @@ Note: requires MongoDB and RabbitMQ to be running to start correctly.
 
     sudo node index.js
 
+ * http://127.0.0.1:80/ui -- MonoRail Web UI
```

Comment:
The default is the one hard coded into http-service.js.

While the one in the config file monorail.json will overwrite the default in http-service.js.